src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/InProcess/SuppressedIssuesFileSynchronizerTests.cs

```diff
@@ -191,18 +188,21 @@ private static BindingConfiguration CreateConnectedConfiguration(string projectK
             ISonarQubeIssuesProvider suppressedIssuesProvider = null,
             ISuppressedIssuesFileStorage suppressedIssuesFileStorage = null,
             IActiveSolutionBoundTracker activeSolutionBoundTracker = null,
-            IThreadHandling threadHandling = null)
+            IThreadHandling threadHandling = null,
+            ILogger logger = null)
         {
             suppressedIssuesMonitor ??= Mock.Of<ISuppressedIssuesMonitor>();
             suppressedIssuesProvider ??= Mock.Of<ISonarQubeIssuesProvider>();
             suppressedIssuesFileStorage ??= Mock.Of<ISuppressedIssuesFileStorage>();
             activeSolutionBoundTracker ??= Mock.Of<IActiveSolutionBoundTracker>();
             threadHandling ??= new NoOpThreadHandler();
+            logger ??= new TestLogger();
```

Comment:
This isn't actually required now - I added it when trying to add tests for the exception handling in for the `void` method.
I could leave it or remove it - no strong preference.